commit 01d386de765e6ffca8267fcee0ded9c44f9c4ef0 (refs/changes/99/2999/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-11-27T10:28:07-08:00 (1 year, 10 months ago)
    
    PUBAPI-1463 test-cloudapi failing in test/images.70.test.js "expected: 'smartmachine'    actual:   'virtualmachine'"

diff --git a/test/images.70.test.js b/test/images.70.test.js
index f92127f..434ac4e 100644
--- a/test/images.70.test.js
+++ b/test/images.70.test.js
@@ -34,24 +34,26 @@ function checkImage(t, image, path) {
     }
 
     t.ok(image, 'image object ok');
-    t.ok(image.id, 'image.id');
+    t.ok(image.id, image.id + ': image.id');
 
-    t.ok(image.name, 'image.name');
-    t.ok(image.version, 'image.version');
-    t.ok(image.type, 'image.type');
-    t.ok(image.requirements, 'image.requirements');
+    t.ok(image.name, image.id + ': image.name');
+    t.ok(image.version, image.id + ': image.version');
+    t.ok(image.type, image.id + ': image.type');
+    t.ok(image.requirements, image.id + ': image.requirements');
 
     t.notEqual(image.name, 'docker-layer',
-        'should be no listed docker-layer images');
+        image.id + ': should be no listed name="docker-layer" images');
 
     if (/\/images/.test(path)) {
-        t.equal(typeof (image.urn), 'undefined', 'image.urn');
+        t.equal(typeof (image.urn), 'undefined', image.id + ': image.urn');
     } else {
-        t.ok(image.urn, 'image.urn');
+        t.ok(image.urn, image.id + ': image.urn');
     }
 
-    t.equal(typeof (image.default), 'undefined', 'image.default');
-    t.equal(image.type, 'smartmachine');
+    t.equal(typeof (image.default), 'undefined', image.id + ': image.default');
+    t.ok(['smartmachine', 'virtualmachine'].indexOf(image.type) !== -1,
+        image.id + ': image.type is one of smartmachine or virtualmachine: '
+        + image.type);
 }
 
 
